### ISSUE TITLE
Version on ROMHacking.net missing DLL

### ISSUE THREAD

**Issue by jonko0493** [Dormanil/UMDReplaceK#2]
After downloading the executable from ROMHacking.net and running it, it returns:

```
The application to execute does not exist: 'C:\Users\jonko\Downloads\UMDReplaceK.dll'.
```

This is because .NET apps are not static exes and need to be packaged with the DLL and any resources built on compilation.

cc/ @SpudManTwo

**Comment by SpudManTwo** [comment]
So firstly, you're right. I think when it got uploaded, it wanted just the exe and didn't include the dlls with it. I'll see if I can get it updated on RomHacking.net

Additionally, I'd recommend leaving issues on my version over on SpudManTwo/UMDReplaceK instead of here on Dormanil's version. His is pretty out of date since it originally stemmed from our collaborative effort but I ended up doing all the more recent updates to it. Thanks for CC'ing me on this. If you want/need, I can go perform a release build over on my github for the project so that way you can access the DLLs.

**Comment by jonko0493** [comment]
Thanks for the quick response!

I actually initially tried going to your repo to post the issue, but it doesn't seem like issues are enabled there:

![image](https://user-images.githubusercontent.com/69772986/166855982-70824e22-414d-438d-a8ab-cea6a6e9e9c4.png)

Regardless, thank you for updating it! I appreciate it!

**Comment by SpudManTwo** [comment]
Yep, no problem. I just enabled Issues (which was turned off by default #ThanksGithub ). Let me know if you encounter anything else
